src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.cs.xlf

```diff
@@ -843,6 +843,46 @@ Obecné přetypování (IL unbox.any) používané sekvencí vrácenou metodou E
         <target state="translated">Nepředávejte literály jako lokalizované parametry</target>
         <note />
       </trans-unit>
+      <trans-unit id="DoNotPassNonNullableValueToArgumentNullExceptionThrowIfNullCodeFixTitle">
+        <source>Do not call 'ArgumentNullException.ThrowIfNull' with a non-nullable value</source>
+        <target state="new">Do not call 'ArgumentNullException.ThrowIfNull' with a non-nullable value</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="DoNotPassNonNullableValueToArgumentNullExceptionThrowIfNullDescription">
+        <source>'ArgumentNullException.ThrowIfNull' throws when the passed argument is 'null'. Certain constructs like non-nullable structs, 'nameof()' and 'new' expressions are known to never be null, so 'ArgumentNullException.ThrowIfNull' will never throw.</source>
+        <target state="new">'ArgumentNullException.ThrowIfNull' throws when the passed argument is 'null'. Certain constructs like non-nullable structs, 'nameof()' and 'new' expressions are known to never be null, so 'ArgumentNullException.ThrowIfNull' will never throw.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="DoNotPassNonNullableValueToArgumentNullExceptionThrowIfNullMessage">
+        <source>Calling 'ArgumentNullException.ThrowIfNull' and passing a non-nullable value does nothing</source>
+        <target state="new">Calling 'ArgumentNullException.ThrowIfNull' and passing a non-nullable value does nothing</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="DoNotPassNonNullableValueToArgumentNullExceptionThrowIfNullTitle">
+        <source>Do not pass a non-nullable value to 'ArgumentNullException.ThrowIfNull'</source>
+        <target state="new">Do not pass a non-nullable value to 'ArgumentNullException.ThrowIfNull'</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="DoNotPassNullableStructToArgumentNullExceptionThrowIfNullCodeFixTitle">
+        <source>Do not call 'ArgumentNullException.ThrowIfNull' with a struct</source>
+        <target state="new">Do not call 'ArgumentNullException.ThrowIfNull' with a struct</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="DoNotPassNullableStructToArgumentNullExceptionThrowIfNullDescription">
```

Comment:
❓ Do we really need this? SharpLab suggests that release builds will not box a non-null value in this case.

Reply:
Can you share an example? I can see the `box` instruction [here](https://sharplab.io/#v2:C4LglgNgPgAgTARgLACgYGYAE9MGFMDeqmJ2WMALJgLIAUYAdsAPyYAeAlMaUSqf5gCCAJwDmAVwC2AUyYA5cRAgBRNgGNpAB2BgA9gwB0AFQAWw3QHcAkgDMFS2pwDc3EgF9UboA===).